ui/src/sources/components/InfluxTable.js

```diff
@@ -0,0 +1,96 @@
+import React, {PropTypes} from 'react'
+import {Link} from 'react-router'
+
+import Dropdown from 'shared/components/Dropdown'
+
+const kapacitorDropdown = (kapacitors) => {
+  if (!kapacitors || kapacitors.length === 0) {
+    return (
+      <span>--</span>
+    )
+  }
+  const kapacitorItems = kapacitors.map((k) => {
+    return {text: k.name}
+  })
+  return (
+    <Dropdown items={kapacitorItems} onChoose={() => {}} selected={kapacitorItems[0].text} />
+  )
+}
+
+const InfluxTable = ({
+  sources,
+  source,
+  handleDeleteSource,
+  location,
```

Comment:
I have nooo idea why I couldn't get `withRouter` to work with this component. When I wrapped the component with it, I wasn't getting `location` added to the props for some reason.